dist/tools/riotboot_gen_hdr/genhdr.c

```diff
@@ -112,3 +112,51 @@ int genhdr(int argc, char *argv[])
 
     return 0;
 }
+
+int updatehdr(int argc, char *argv[])
+{
+    if (argc < 2) {
+        fprintf(stderr, "usage: genhdr update <file> <new_version>\n");
+        return -1;
+    }
+    const char *file = argv[1];
+
+    riotboot_hdr_t hdr = { 0 };
+    int res = from_file(file, &hdr, sizeof(hdr));
+    if (res < (int)sizeof(hdr)) {
+        fprintf(stderr, "Can't read header from %s\n", file);
+        return -EIO;
+    }
+
+    if (hdr.magic_number != RIOTBOOT_MAGIC) {
+        fprintf(stderr, "Invalid magic: %x\n", hdr.magic_number);
+        return -EIO;
+    }
+
+    hdr.version = atoi(argv[2]);
```

Comment:
I guess there's no sensible default. :/

Reply:
If we want to update the version number it makes sense that we have to specify a version number.